src/Native/System.Net.Security.Native/pal_gssapi.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "pal_gssapi.h"
+#include <assert.h>
+#include <string.h>
+
+static_assert(PAL_GSS_COMPLETE == GSS_S_COMPLETE, "");
+static_assert(PAL_GSS_CONTINUE_NEEDED == GSS_S_CONTINUE_NEEDED, "");
+
+static char gss_mech_value[] = "\x2b\x06\x01\x05\x05\x02";
+
+static gss_OID_desc gss_mech_spnego_OID_desc = {6, static_cast<void*>(gss_mech_value)};
+
+static gss_OID gss_mech_spnego_OID = &gss_mech_spnego_OID_desc;
+
+static gss_OID_set_desc gss_mech_spnego_OID_set_desc = {1, &gss_mech_spnego_OID_desc};
+
+static gss_OID_set gss_mech_spnego_OID_set = &gss_mech_spnego_OID_set_desc;
```

Comment:
Static initialization and destruction can be dangerous since the loading and unloading of libc (and other libraries) is non-deterministic. Are there ways to do this without all these statics?